src/ext/property-panel/data.ts

```diff
@@ -26,6 +28,69 @@ function isTotalsVisible(itemData: EngineAPI.IHyperCubeDimensionDef, _: unknown,
   return idx === 0 || idx >= noOfLeftDims;
 }
 
+// TODO: scope this out to common repo, don't see any differences in properties of default values atm
+const columnResize = {
+  type: {
+    type: "string",
+    component: "dropdown",
+    ref: "qDef.columnWidth.type",
+    translation: "Object.Table.Column.Width",
+    options: [
+      {
+        value: ColumnWidthType.Auto,
+        translation: "Common.Auto",
+      },
+      {
+        value: ColumnWidthType.FitToContent,
+        translation: "Object.Table.Column.FitToContent",
+      },
+      {
+        value: ColumnWidthType.Pixels,
+        translation: "Object.Table.Column.Pixels",
+      },
+      {
+        value: ColumnWidthType.Percentage,
+        translation: "Object.Table.Column.Percentage",
+      },
+    ],
+    defaultValue: ColumnWidthType.Auto,
+  },
+  sizePixels: {
```

Comment:
I'm getting some unexpected result (for me) when switching to pixel based size via PP. The default value is displayed in the PP but the layout is not actually updated with the value. Only the `type` property exist on `qMeasureInfo.columnWidth`.

I'd say it works fine because ther is likely some fallback logic when resolving the pixel based column width. Still worth noting I think.

Reply:
yeah good point, I'll look at that. Maybe we want to set the value for pixels when changing the type to pixels

Reply:
will scope it out of this PR in any case. I can do it as I add the disclaimer